.husky/pre-commit

```diff
@@ -4,4 +4,4 @@
 echo "\nRunning GIT hooks..."
 yarn cleanbuild
 yarn nx affected --target=lint
-yarn nx affected --target=test
\ No newline at end of file
+# yarn nx affected --target=test
```

Comment:
I think we should not commit commenting this out
was instructed this earlier so I think will be better to remove the comment from it

Reply:
okay